next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,262 @@
+---
+description: Easily integrate with your exchange, custody solution, or product using the IOTA SDK.
+sidebar_label: Integrate IOTA SDK With Your Exchange
+keywords:
+  - integrate
+  - exchange
+  - account model
+  - addresses
+  - IOTA SDK
+  - setup
+  - NodeJS
+  - explanation
+---
+
+import WarningUnlockConditions from '../_admonitions/_warning-unlock-conditions.md';
+
+# Stardust Exchange Integration Guide
+
+:::note
+
+The Stardust update is a set of protocol changes that were first deployed to the Shimmer network.
+See [What is Stardust?
+](https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust/) for an overview of its features or [TIPs](https://github.com/iotaledger/tips) flagged `Stardust` for a deeper understanding of the protocol changes.
+
+:::
+
+:::warning
+
+The Stardust update allows for complex tokenization schemes that can lead to monetary losses if misused.
+Transaction outputs may have multiple unlocking conditions which may require returning some or all of the amount, which could expire if not claimed in time, or which may not be unlockable for a very long time. You can lose money if you do not check the unlock conditions before accepting a deposit!
```

Comment:
Using multiple times `which` made me confused a little, Maybe smth  like this: 
```suggestion
Transaction outputs might have multiple unlocking conditions. These conditions may require the return of part or all of the transaction amount, have an expiration date if not claimed on time, or they may not be unlockable for an extended period of time. Therefore, you need to review these unlock conditions carefully before accepting any deposits to avoid loss of funds.
```